client/src/app/site/motions/components/motion-list/motion-list.component.ts

```diff
@@ -179,6 +201,34 @@ export class MotionListComponent extends ListViewBaseComponent<ViewMotion> imple
      * @param multiselectPromise The promise returned by multiselect actions.
      */
     public multiselectWrapper(multiselectPromise: Promise<void>): void {
-        multiselectPromise.then(() => this.toggleMultiSelect(), this.raiseError);
+        const me = this;
+        multiselectPromise.then(() => {
+            me.checkSelection();
+        }, error => {
+            me.raiseError(error);
+            me.checkSelection();
+        });
+
+    }
+
+    /**
+     * returns true if there are any categories available
+     */
+    public get hasCategories(): boolean {
+        return this.categoryRepo.getViewModelList().length > 0;
```

Comment:
This function is not used. Please remove and also `hasTags()` and `hasMotionBlocks()`.